networking/src/ApiClient.ts

```diff
@@ -623,11 +622,11 @@ type AWSTeamAnswer = {
     currentState: GameSessionState
     currentQuestionIndex: number
     questionId: string
-    teamMemberId: string
+    teamMemberAnswersId: string
     text: string
     answer: string
-    confidenceLevel: ConfidenceLevel
-    hint: string
+    confidenceLevel?: ConfidenceLevel | null
```

Comment:
setting these as optional in the schema as they are not created when a `answer` is originally created